lib/copy_peste/modules.rb

```diff
@@ -0,0 +1,3 @@
+require_relative 'modules/base.rb'
```

Comment:
I assume that's a `Base` class. That's how OO abstract classes are made in Ruby.